core/eval.go

```diff
@@ -863,6 +863,51 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+func evalCOMMANDLIST(args []string) []byte {
+	return RESP_OK
+}
+func evalCOMMANDCOUNT(args []string) []byte {
+	return RESP_OK
+}
+func evalCOMMANDINFO(args []string) []byte {
+	return RESP_OK
+}
+func evalCOMMANDDOCS(args []string) []byte {
+	return RESP_OK
+}
+func evalCOMMANDGETKEYS(args []string) []byte {
+	return RESP_OK
+}
+
+// evalCOMMAND returns info of all the commands in Dice when no args are provided
+// When args are provided:
+// LIST: return lists all the Dice commands.
+// COUNT: return total count of commands in Dice.
+// INF0: returns info of command(s).
+// DOCS: returns docs of command(s).
+// GETKEYS: returns keys from a full Dice command.
+func evalCOMMAND(args []string) []byte {
+	if len(args) == 0 {
+		return evalCOMMANDINFO(args)
+	}
+
+	subcommand := args[0]
+	switch subcommand {
+	case "LIST":
+		return evalCOMMANDLIST(args)
+	case "COUNT":
+		return evalCOMMANDCOUNT(args)
+	case "INFO":
+		return evalCOMMANDINFO(args)
+	case "DOCS":
+		return evalCOMMANDDOCS(args)
+	case "GETKEYS":
+		return evalCOMMANDGETKEYS(args)
+	default:
+		return Encode(fmt.Errorf("ERR unknown subcommand '%s'. Try COMMAND HELP", subcommand), false)
```

Comment:
this error message is taken from Redis
<img width="767" alt="Screenshot 2024-07-15 at 18 56 25" src="https://github.com/user-attachments/assets/bcc2ff87-698e-4825-9e1a-36ae3dfc614c">